esiclient/v1/node_network.py

```diff
@@ -264,21 +264,40 @@ def take_action(self, parsed_args):
         ironic_client = self.app.client_manager.baremetal
         neutron_client = self.app.client_manager.network
 
-        node = None
-        port = None
+        current_node = ironic_client.node.get(node_uuid)
 
-        with concurrent.futures.ThreadPoolExecutor() as executor:
-            f1 = executor.submit(ironic_client.node.get, node_uuid)
-            f2 = executor.submit(neutron_client.find_port, port_uuid)
-            node = f1.result()
-            port = f2.result()
-
-        if not port:
+        if current_node is None:
             raise exceptions.CommandError(
-                "ERROR: Port {1} not attached to node {0}".format(
-                    node.name, port_uuid))
+                "ERROR: The node {0} is not available".format(
+                    node_uuid))
```

Comment:
I don't think you actually need this check, simply because `ironic_client.node.get` already returns an error message if the node is not available; you may want to test to be sure, but there's a reason why this check didn't exist earlier.